core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -1071,6 +1088,14 @@ object IO {
   final def parAll[E, A](as: Iterable[IO[E, A]]): IO[E, List[A]] =
     parTraverse(as)(identity)
 
+  /**
+   * Evaluate each effect in the structure in parallel, and collect
+   * the results. Only up to `n` tasks run in parallel.
+   * This is a version of `parAll`, with a throttle.
+   */
+  final def parAllThrottled[E, A](n: Long)(as: Iterable[IO[E, A]]): IO[E, List[A]] =
```

Comment:
Let's call this: `sequenceParN` and introduce an alias `collectParN`.

Reply:
done